Jenkinsfile

```diff
@@ -30,28 +30,29 @@ pipeline {
-                             // example code / documentation
-                             ',**/org/eclipse/jetty/embedded/**' +
-                             ',**/org/eclipse/jetty/asyncrest/**' +
-                             ',**/org/eclipse/jetty/demo/**' +
-                             // special environments / late integrations
-                             ',**/org/eclipse/jetty/gcloud/**' +
-                             ',**/org/eclipse/jetty/infinispan/**' +
-                             ',**/org/eclipse/jetty/osgi/**' +
-                             ',**/org/eclipse/jetty/http/spi/**' +
-                             // test classes
-                             ',**/org/eclipse/jetty/tests/**' +
-                             ',**/org/eclipse/jetty/test/**',
-                     execPattern: '**/target/jacoco.exec',
-                     classPattern: '**/target/classes',
-                     sourcePattern: '**/src/main/java'
-              recordIssues id: "jdk17", name: "Static Analysis jdk17", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), errorProne(), spotBugs(), javaDoc()]
+//              jacoco inclusionPattern: '**/org/eclipse/jetty/**/*.class',
+//                     exclusionPattern: '' +
+//                             // build tools
+//                             '**/org/eclipse/jetty/ant/**' +
+//                             ',*/org/eclipse/jetty/maven/its/**' +
+//                             ',**/org/eclipse/jetty/its/**' +
+//                             // example code / documentation
+//                             ',**/org/eclipse/jetty/embedded/**' +
+//                             ',**/org/eclipse/jetty/asyncrest/**' +
+//                             ',**/org/eclipse/jetty/demo/**' +
+//                             // special environments / late integrations
+//                             ',**/org/eclipse/jetty/gcloud/**' +
+//                             ',**/org/eclipse/jetty/infinispan/**' +
+//                             ',**/org/eclipse/jetty/osgi/**' +
+//                             ',**/org/eclipse/jetty/http/spi/**' +
+//                             // test classes
+//                             ',**/org/eclipse/jetty/tests/**' +
+//                             ',**/org/eclipse/jetty/test/**',
+//                     execPattern: '**/target/jacoco.exec',
+//                     classPattern: '**/target/classes',
+//                     sourcePattern: '**/src/main/java'
+              recordIssues (id: "analysis-jdk17", name: "Static Analysis jdk17", aggregatingResults: true, enabledForFailure: true, tools: [mavenConsole(), java(), checkStyle(), errorProne(), spotBugs(), javaDoc()])
+              recordCoverage(name: "Coverage jdk17", id: "coverage-jdk17", tools: [[parser: 'JACOCO']], sourceCodeRetention: 'MODIFIED', sourceDirectories: [[path: 'src/main/java'], [path: 'target/generated-sources/ee8']])
```

Comment:
Can we still have the inclusion / exclusion patterns?

Reply:
In fact those exclusions are already defined in the pom https://github.com/jetty/jetty.project/blob/67c5fbcd60a9d3843f024b60f8edd1e12a359e29/pom.xml#L1962

so datas are not generated which makes this configuration a bit of no effect/redundant and not really in sync